LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2016 Kamil Filipowski,
+                    Maciej Michalik,
+                    Bernard Pietraga,
+                    Alicja Raszkowska,
+                    Karolina Rostek,
+                    Katarzyna Sikorska,
+                    Piotr Sordyl
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
```

Comment:
MIT license was taken from https://opensource.org/licenses/MIT, so there should not be any mistakes